traefik/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
-version: 10.1.6
+version: 10.1.7
```

Comment:
Please update the chart and Traefik version.

Reply:
done